app/src/main/java/de/westnordost/streetcomplete/quests/max_height/AddMaxHeight.kt

```diff
@@ -34,7 +36,14 @@ class AddMaxHeight : OsmElementQuestType<MaxHeightAnswer> {
     """.toElementFilterExpression() }
 
     private val bridgeFilter by lazy { """
-        ways with (highway or railway) and bridge and layer
+        ways with (
+            highway ~ ${(ALL_ROADS + ALL_PATHS).joinToString("|")}
+            or railway ~ rail|light_rail|subway|narrow_gauge|tram
```

Comment:
`railway=disused` also should be fine

Reply:
https://taginfo.openstreetmap.org/keys/railway#values `railway=preserved|miniature|monorail|funicular may be too exotic to bother with them or worth including